raiden-ts/src/transfers/epics/utils.ts

```diff
@@ -108,3 +111,48 @@ export function matchWithdraw<
           'totalWithdraw' in data ? data.totalWithdraw : data.total_withdraw,
         )));
 }
+
+/**
+ * @param channel - Channel state
+ * @param config - Raiden config
+ * @returns FeeSchedule for channel
+ */
+export function getFeeScheduleForChannel(
+  channel: Pick<Channel, 'token'>,
+  config: Pick<RaidenConfig, 'tokenFees'>,
+): FeeSchedule {
+  const tokenSchedule = {
+    flat: Zero as Int<32>,
+    ...config.tokenFees[AddressZero],
+    ...config.tokenFees[channel.token],
+  };
+  // split per-token feeSchedule in half, one for each channel (in & out)
```

Comment:
This is code deduplication (as required by codeclimate's quality analysis. Otherwise I'd have to make two interfaces exactly the same: `TokenFeeSchedule` and `ChannelFeeSchedule`. This is the same behavior from the reference implementation (PC), with the difference that they spread this "decision" in several places, while I made a single utility function for that.